src/app/wallet/settings/modal/help-modal/console-modal.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="containerX">
```

Comment:
Duplicate?

Reply:
Yeah, We have decided to have the same inside setting for future perspective as in future this will moves back to setting page.